### PR TITLE
Refactor main.go

### PR DESCRIPTION
**Pull request by dontlaugh**
- Move defaultBootstrapConfig to function scope
- Move func main to the top, separate out flags

### CODE REVIEW

**Comment by dontlaugh** [comment]
We'll use rebase to keep these commits separate

File: main.go [hunk 1]
```diff
@@ -73,18 +60,32 @@ func init() {
...
+		os.Exit(1)
+	}
+}
+
 func run() error {
-	flag.StringVar(&configFile, "config", "", "The operator will load its initial configuration from this file if defined.")
-	flag.BoolVar(&development, "development", false, "Run in development mode.")
+	flag.Parse()
+
+	flag.StringVar(&configPath, "configPath", "", "The operator will load its initial configuration from this file if defined.")
```
Comment (bcmendoza) [inline main.go:79]: You’ll need to update the manifest as well.

**Review by bcmendoza — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (dontlaugh) [reply main.go:79]: Alec asked me to run `make generate manifests`. It yielded no diff.

**Review by alecholmez — APPROVED** [review]
Nice this looks good

**Pull request merged**